src/site/content/en/authors/individual.njk

```diff
@@ -19,9 +19,39 @@ pagination:
   <web-actions actions="subscribe"></web-actions>
 
   <div class="w-layout-container w-pb--non">
-    <header class="w-page-header">
-      <h1 class="w-page-header__headline">{{ renderData.title }}</h1>
-      <p class="w-page-header__copy">{{ renderData.description | safe }}</p>
+    <header class="w-page-header w-author-page-header">
+      {% if paged.data.hero %}
+        <div class="w-author-page__image--container">
+          <img src="{{paged.data.hero}}" alt="{{paged.data.alt}}" class="w-author-page__image">
+        </div>
+      {% endif %}
+      <div class="w-author-page__description">
+        <h1 class="w-page-header__headline">{{ renderData.title }}</h1>
+        <p class="w-page-header__copy">{{ renderData.description | safe }}</p>
+        {% if paged.homepage %}
```

Comment:
I think you may want to put all of these anchors into a containing div and apply the margin to the anchors instead of the images within the anchors. You'll probably want to use flexbox on the container. Something like:

```
<div class="icon-list">
  <a class="icon-list__icon"><img/></a>
</div>
```